tests/tests_ethtool_features_initscripts.yml

```diff
@@ -1,14 +1,17 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
 ---
-# set network provider and gather facts
 - hosts: all
+  name: Run playbook 'playbooks/tests_ethtool_features.yml' with initscripts as provider
   tasks:
     - include_tasks: tasks/el_repo_setup.yml
     - name: Set network provider to 'initscripts'
       set_fact:
         network_provider: initscripts
+      tags:
+        - always
 
-# workaround for: https://github.com/ansible/ansible/issues/27973
-# There is no way in Ansible to abort a playbook hosts with specific OS
-# releases Therefore we include the playbook with the tests only if the hosts
-# would support it.
 - import_playbook: playbooks/tests_ethtool_features.yml
+  when: ansible_distribution in ['CentOS','RedHat'] and
+    ansible_distribution_major_version | int < 9 or
+    ansible_distribution == 'Fedora'
```

Comment:
Please group the conditions with parentheses

Reply:
grouped